kogito-business-rules-quarkus/README.md

```diff
@@ -68,8 +98,26 @@ To make use of this application it is as simple as putting a sending request to
 
 ```
 
-Complete curl command can be found below:
+Complete curl command can be found below for adults:
 
 ```
 curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"person" : {"name" : "john", "age" : 20}}' http://localhost:8080/persons
 ```
+
+After the Curl command you should see a similar console log
+```
+{"id":"fd4f629d-6822-4ca2-a8a6-a74f5f81e83d","person":{"name":"john","age":20,"adult":true}} 
+```
+
+Complete curl command can be found below for children:
+
+```
+curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"person" : {"name" : "john", "age" : 5}}' http://localhost:8080/persons
+```
+
+After the Curl command you should see a similar console log
+
+```
+{"id":"c59054b9-aa1d-4771-bc5e-40f8b32d3ff5","person":{"name":"john","age":5,"adult":false}} 
+```
+
```

Comment:
We should also show here the ChildrenHandling user task for a child:

`curl http://localhost:8080/persons/{uuid}/tasks` should return something like `{"f726a3ca-f51d-4c44-98c0-1cdcc74805b4":"ChildrenHandling"}`

=> Replace `{uuid}` with persons id

Then 
`curl http://localhost:8080/persons/{uuid}/ChildrenHandling/{tuuid}` should return `{"person":{"name":"john","age":5,"adult":false},"id":"f726a3ca-f51d-4c44-98c0-1cdcc74805b4","name":"ChildrenHandling"}`

=> Replace `{uuid}` with persons id and `{tuuid}` with task id

Then we can validate the child:

`curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{}' http://localhost:8080/persons/{uuid}/ChildrenHandling/{tuuid}` should return `{"id":"09f98756-b273-4ceb-9308-fae7cc423904","person":{"name":"john","age":5,"adult":false}}`

=> Replace `{uuid}` with persons id and `{tuuid}` with task id